include/Server.hpp

```diff
@@ -0,0 +1,55 @@
+#ifndef SERVER_HPP
+# define SERVER_HPP
+# include <vector>
+# include <iostream>
+# include <sys/socket.h>
+# include <sys/types.h>
+# include <sys/event.h>
+# include <sys/time.h>
+# include <stdlib.h>
+# include <unistd.h>
+# include <netdb.h>
+# include <signal.h>
+# include "SockConnect.hpp"
```

Comment:
can you include header and do forward declaration?

Reply:
Yes as previous comment, I will modify these double header inclusions in the new file. (I will push it by new branch)